apps/docs/src/app/core/component-docs/form-message/examples/form-message-example.component.html

```diff
@@ -3,7 +3,13 @@
         Message With Input
     </label>
     <fd-form-input-message-group>
-        <input fd-form-control type="text" id="input-message" placeholder="Field placeholder text" [state]="'information'" />
+        <input 
+            fd-form-control 
+            type="text" 
+            id="input-message" 
+            placeholder="Field placeholder text" 
+            [state]="'information'" 
```

Comment:
```suggestion
            state="information" 
```
please check all file on unnecessary binding